test_unstructured/unit_utils.py

```diff
@@ -1,13 +1,42 @@
 """Utilities that ease unit-testing."""
 
+from __future__ import annotations
+
 import datetime as dt
 import difflib
 import pathlib
-from typing import List, Optional
+from typing import Any, List, Optional
+from unittest.mock import (
+    ANY,
+    MagicMock,
+    Mock,
+    PropertyMock,
+    call,
+    create_autospec,
+    mock_open,
+    patch,
+)
+
+from pytest import FixtureRequest, LogCaptureFixture  # noqa: PT013
 
 from unstructured.documents.elements import Element
 from unstructured.staging.base import elements_from_json, elements_to_json
 
+__all__ = (
+    "ANY",
+    "FixtureRequest",
+    "LogCaptureFixture",
+    "MagicMock",
+    "Mock",
+    "call",
+    "class_mock",
+    "function_mock",
+    "initializer_mock",
+    "instance_mock",
+    "method_mock",
+    "property_mock",
+)
+
```

Comment:
Not that this needs to be changed, but isn't `__all__` used to define what's imported when you use * to import, or is there another use?

Reply:
Yes, it is, but it also indicates what you are intentionally "exporting" and prevents ruff errors related to "xyz is imported but not used". That's why I have it here.